release notes/v1.0.0-rc1.md

```diff
@@ -0,0 +1,186 @@
+INFO[0001] https://quickpizza.grafana.com/images/pizza.png  source=console
+...
+```
+
+### `k6/experimental/webcrypto` promoted to stable and available globally under `crypto` [#4278](https://github.com/grafana/k6/pull/4278)
+
+With this release, the `k6/experimental/webcrypto` module is promoted to stable and available globally under `crypto`. So you don't need to import it anymore.
+
+```javascript
+export default function () {
+  const myUUID = crypto.randomUUID();
+
+  console.log(myUUID);
+}
+```
+
+`k6/experimental/webcrypto` is deprecated and will be removed in `v1.1.0`.
+
+### Support for custom templates in `k6 new` command [#4618](https://github.com/grafana/k6/pull/4618)
+
+The `k6 new` command now accepts a path to a file to use as a template for the new script. Templates use [Go templates syntax](https://pkg.go.dev/text/template#pkg-overview) and can include the following variables:
+
+- `ScriptName`: The name of the new script.
+- `ProjectID`: The ID of the Grafana Cloud project to use for the new script.
+
+To generate a new script using a custom template, use the following command:
+
+```sh
+k6 new --template /path/to/my-template.js
+```
+### Secret Sources [#4514](https://github.com/grafana/k6/pull/4514), [#4621](https://github.com/grafana/k6/pull/4621)
+
+Now k6 has a way to get secrets from different sources with a way to extend that. Among other things this means that the values received from a secret source will be redacted from the logs.
+
+At this point the two implementations included are to read them from key-value file or from cli flags, are meant mostly for testing the feature.
+
+It already has extension support that can be used to implement retrieving secrets from more secure sources.
+
+In the future k6 will likely add additional implementations that are more production ready.
+
+## UX improvements and enhancements
+
+- [#4547](https://github.com/grafana/k6/pull/4547) The k6 banner now outputs with the original TrueColor (24-bit) logo only if the terminal supports it.
+- [#4590](https://github.com/grafana/k6/pull/4590) moves a docker-compose example with InfluxDB to the `examples/docker-compose` directory and adds an opentelemetry example.
+- [#4602](https://github.com/grafana/k6/pull/4602), [#4629](https://github.com/grafana/k6/pull/4629) Better error message on options error from script. k6 now will try to print the part of the options that fails as JSON.
+
+## Bug fixes
+
+- [#4544](https://github.com/grafana/k6/pull/4544) Fix race in ReadableStream.cancel and run WPT test with race detection for easier finding of similar problems.
+- [#4567](https://github.com/grafana/k6/pull/4567), [#4558](https://github.com/grafana/k6/pull/4558), [#4574](https://github.com/grafana/k6/pull/4574), [#4610](https://github.com/grafana/k6/pull/4610) Fixes race conditions with the browser module's `ElementHandle`, `Mouse`, `Keyboard`, and `Response`.
+
+## Maintenance and internal improvements
```

Comment:
How about trying to combine the items when it's possible, trying to keep essential for users on top of the list, what was discussed in our past retro.

Like, everything that related to the test or running the test in CI could go down and combined under the single list item. WDYT?

Reply:
Done